src/main/webapp/WEB-INF/templates/search.jsp

```diff
@@ -12,7 +14,25 @@
     <t:base/>
     <div class="gcse-searchbox-only" data-resultsUrl="/search"></div>
     <div class="srp-container">
-      <div class="gcse-searchresults-only"></div>
+      <div class="srp-child">
+        <div class="gcse-searchresults-only"></div>
+      </div>
+      <div class="srp-child">
+        <%ArrayList<Card> cards = (ArrayList<Card>)request.getAttribute("card_list");%>
```

Comment:
It's not hard to make this a for loop. I did it in the example code, see https://github.com/googleinterns/step1-2020/commit/8a66c41c2ebe9e3ae0000b6b800a40098b801f90#diff-de7f7adba792bdbbac028ec11fe4dc91

It also avoids having to import and declare an `ArrayList`. (I should point out that you don't need an ArrayList here, just a List, and that's why you're being forced to cast - because it's only a List.)